src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -35,77 +37,159 @@
+		
+		public static class Network {
+			public static final String VISION_GOAL_NETWORK_TABLE_ADDRESS = "GRIP/myContoursReport";
+		}
+		
+		public static class Component {}
+		
 		public static class Pneumatics {}
+		public static int lidarTurnP = 0;// put these in the constants section of RobotMap
+		public static int lidarTurnI = 0;
+		public static int lidarTurnD = 0;
 	}
 	
 	public static class Component {
 		public static PDP pdp;
-		public static Chassis chassis;
-		public static PositionEncodedMotor leftWheel;
-		public static PositionEncodedMotor rightWheel;
+		public static PositionSensorMotor leftWheel;
+		public static PositionSensorMotor rightWheel;
 		public static CustomEncoder leftWheelEncoder;
 		public static CustomEncoder rightWheelEncoder;
-		public static Flywheel flywheel;
-		public static Dump ballDumper;
-		public static BallInnie ballIntake;
+		public static CustomEncoder elevatorEncoder;
+		public static Chassis chassis;
+		public static NavX navX;
+		public static CustomPIDController chassisDrivePID;
+		public static BallIO ballIO;
+		public static SolenoidShifters gearSlotOpener;
+		public static Servo gearFlap;
 		public static Subsystem[] mainSubsystems;
 		public static CustomXbox driverXbox;
 		public static CustomJoystick operatorStick;
+		// lidar
+		public static CustomPIDController lidarMC;
+		public static CANEncoder lidarTurnEncoder;
+		public static CapSpeedModifier lidarCap;
 		// Vision
-		public static AligningCamera alignCamera;
+	}
+	
+	public static class HumanInput {
+		public static class Driver {
+			public static CustomXbox xbox;
+		}
+		
+		public static class Operator {
+			public static CustomJoystick stick;
+		}
 	}
 	
 	public RobotMap() {
 		Component.pdp = new PDP();
 		// Chassis
 		Component.leftWheelEncoder = new CANEncoder(Port.CAN.leftEncoder);
 		Component.leftWheelEncoder.setReverseDirection(true);
-		Component.leftWheel = new PositionEncodedMotor("leftWheel", new AccelerationCap(Component.pdp), new CustomPIDController(Component.leftWheelEncoder), new VictorSP(Port.PWM.leftDriveMotor));
-		Component.leftWheel.disablePID(); // TODO add encoders
-		// Component.leftWheel.setInverted(true);
+		Component.leftWheel = new PositionSensorMotor("leftWheel", new AccelerationCap(Component.pdp), new CustomPIDController(Component.leftWheelEncoder), new VictorSP(Port.PWM.leftDriveMotor));
+		Component.leftWheel.disableMotionController(); // TODO add encoders
 		Component.rightWheelEncoder = new CANEncoder(Port.CAN.rightEncoder);
-		Component.rightWheel = new PositionEncodedMotor("rightWheel", new AccelerationCap(Component.pdp), new CustomPIDController(Component.rightWheelEncoder), new VictorSP(Port.PWM.rightDriveMotor));
-		Component.rightWheel.disablePID(); // TODO add encoders
+		Component.rightWheel = new PositionSensorMotor("rightWheel", new AccelerationCap(Component.pdp), new CustomPIDController(Component.rightWheelEncoder), new VictorSP(Port.PWM.rightDriveMotor));
```

Comment:
Is there a reason to use PositionSensorMotor and PID controller if it's going to be disabled?